lib/pebble.js

```diff
@@ -104,9 +105,23 @@ function addExtraData (first, req, sbx) {
     }
   }
 
+  function addPumpIOB () {
+    if (req.pumpiob) {
+      var result = pumpiob.getPumpIOB(
+        data,
+        sbx.withExtendedSettings(pumpiob).extendedSettings,
+        Date.now()
+      );
+      if (result) {
+        first.iob = result.pebbleDisplay;
+      }
+    }
+  }
+
   addDelta();
   addBattery();
   addIOB();
+  addPumpIOB();
```

Comment:
not sure we should be adding something new to `/pebble`, this thing is so fragile with all the random pebble watchface versions and other apps pulling from it